src/main/webapp/script.js

```diff
@@ -160,6 +160,57 @@ async function processImage(image) {
   drawImageOnCanvas(blurredImage.object, outputCanvas);
 
   updateDownloadButton(blurredImage);
+
+  outputCanvas.onclick = (event) => {
+    tryToUnblurArea(event, image);
+  };
+}
+
+/**
+ * Function to handle click on output canvas
+ * and toggle blurness of rects to blur
+ * if any were clicked.
+ * @param {MouseEvent} event
+ * @param {ImageObject} image
+ */
+function tryToUnblurArea(event, image) {
+  const outputCanvas = document.getElementById('output-canvas');
+
+  // coordinates of click.
+  const eventX = event.offsetX;
+  const eventY = event.offsetY;
+
+  for (const blurRect of image.blurAreas) {
+    const scaleForCanvas = (propertyToScale) => {
```

Comment:
Could you add a comment explaining what this function does? What means propertyToScale?